### ISSUE TITLE
bamCoverage sorting issue

### ISSUE THREAD

**Issue by radaniba** [fidelram/deepTools#95]
Hello

I am recieving an error message and am not finding any similar issue here.

The log says 

```
TASK_8_c_CX5461-RAD51-1_S8.bg is not case-sensitive sorted at line 455646.  Please use "sort -k1,1 -k2,2n" with LC_COLLATE=C,  or bedSort and try again.
```

The script is like : 

```
#! /bin/bash
bamCoverage --bam results/gc_corrected_bam/TASK_6_c_2_CX5461-RAD51-1_S8_gc_bias_corrected.sorted.bam --outFile
Name results/read_density/TASK_8_c_CX5461-RAD51-1_S8 --fragmentLength 250 --normalizeTo1x 3114492550
``` 

any idea ?

**Comment by radaniba** [comment]
do we need to sort the bam file after correctGCBias step ?

**Comment by fidelram** [comment]
The problem with the sorting error is addressed in commit 885aca48747c0b5da5ade35544008d2dc7c73684 Simply install the latest version of deepTools:

pip install --upgrade git+git://github.com/fidelram/deepTools

Yo don't need to sort the bam after correctGCBias.

**Comment by radaniba** [comment]
Thanks @fidelram
